hypershift-operator/controllers/machineconfigserver/machineconfigserver_controller.go

```diff
@@ -113,8 +113,16 @@ func (r *MachineConfigServerReconciler) Reconcile(ctx context.Context, req ctrl.
 			return ctrl.Result{}, err
 		}
 
-		if err := r.Delete(ctx, ignitionRoute); err != nil && !apierrors.IsNotFound(err) {
-			return ctrl.Result{}, err
+		switch mcs.Spec.IgnitionService.Type {
+		case hyperv1.NodePort:
+			r.Log.Info("No extra components to delete in node port strategy")
+		case hyperv1.Route:
+			r.Log.Info("Removing ignition route")
```

Comment:
Nit: log after the deletion and provide context about what was deleted

Reply:
done